RationaleMCP/ReadMe.md

```diff
@@ -15,49 +15,50 @@ After a release is accepted, follow the [check-list for a new release](NewReleas
-|Added in [3.5](https://github.com/modelica/ModelicaSpecification/releases/tag/v3.5)|0033|Annotations for Predefined Plots|([MCP/0033](https://github.com/modelica/ModelicaSpecification/tree/MCP/0033/RationaleMCP/0033))|
-|Active|0032|Selective Model Extension|([MCP/0032](https://github.com/modelica/ModelicaSpecification/tree/MCP/0032/RationaleMCP/0032))|
-|Active|0031|Flat Modelica and MLS modularization|([MCP/0031](https://github.com/modelica/ModelicaSpecification/tree/MCP/0031/RationaleMCP/0031))|
-|On hold|0030|IsClocked Operator|([#2238](https://github.com/modelica/ModelicaSpecification/issues/2238))|
-|Active|0029|License Export|([MCP/0029](https://github.com/modelica/ModelicaSpecification/tree/MCP/0029/RationaleMCP/0029))|
-|Added in [3.4](https://github.com/modelica/ModelicaSpecification/releases/tag/v3.4)|0028|Record Derivatives mixing Real and non-Real|([#2137](https://github.com/modelica/ModelicaSpecification/issues/2137))|
-|Active|0027|Units of Literal Constants|([#2127](https://github.com/modelica/ModelicaSpecification/issues/2127))|
-|Added in [3.4](https://github.com/modelica/ModelicaSpecification/releases/tag/v3.4)|0026|Arc-only Ellipse|([#2045](https://github.com/modelica/ModelicaSpecification/issues/2045))|
-|On hold|0025|Functions with input output|([#2012](https://github.com/modelica/ModelicaSpecification/issues/2012))|
-|Added in [3.4](https://github.com/modelica/ModelicaSpecification/releases/tag/v3.4)|0024|Initialization of Clocked States|([#2007](https://github.com/modelica/ModelicaSpecification/issues/2007))|
-|Added in [3.4](https://github.com/modelica/ModelicaSpecification/releases/tag/v3.4)|0023|Model to Record|([#1953](https://github.com/modelica/ModelicaSpecification/issues/1953))|
-|Added in [3.4](https://github.com/modelica/ModelicaSpecification/releases/tag/v3.4)|0022|Integer to Enumeration|([#1842](https://github.com/modelica/ModelicaSpecification/issues/1842))|
-|Active|0021|Component iterators|([MCP/0021](https://github.com/modelica/ModelicaSpecification/tree/MCP/0021/RationaleMCP/0021))|
-|Added in [3.4](https://github.com/modelica/ModelicaSpecification/releases/tag/v3.4)|0020|Model as Arguments to Functions||
-|Added in [3.4](https://github.com/modelica/ModelicaSpecification/releases/tag/v3.4)|0019|Improvement of Flattening Description|([#1829](https://github.com/modelica/ModelicaSpecification/issues/1829))|
-|Added for [3.5](https://github.com/modelica/ModelicaSpecification/releases/tag/v3.5)|0018|Change specification format||
-|Active|0017|Portable import of FMUs||
-|Active|0016|Semantic Versions||
-|Active|0015|Language Version Header|([MCP/0015](https://github.com/modelica/ModelicaSpecification/tree/MCP/0015/RationaleMCP/0015))|
-|Added in [3.4](https://github.com/modelica/ModelicaSpecification/releases/tag/v3.4)|0014|Conversion|([#1622](https://github.com/modelica/ModelicaSpecification/issues/1622))|
+|Status|Number|Name|Branch|Issue|
```

Comment:
I saw your comment about the use of _Issue_, but I think a good reason to use _Main PR of MCP_ instead is the pedagogical value of explaining that each MCP needs to have a main PR.

Would it also be useful to have a column showing which GitHub label to use for all associated issues and PRs (such as _MCP_ or _MCP0039_)?

Reply:
By the way, if the _Main PR of MCP_ happened to be abused for something else, it wouldn't be the end of the world.

Reply:
The problem with Main PR of MCP is that many MCPs only have a discussion-Issue so far.
Not including them would be a downgrade and two columns (Issue vs Main PR) would cause more problems.